styles.css

```diff
@@ -476,5 +488,240 @@ input:focus {
 .home-indicator {
   width: 134px;
   border-radius: 10px;
-  margin-top: 100px;
+  margin-top: 30px;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  html {
+    box-sizing: border-box;
+  }
+
+  .container {
+    width: auto;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .head {
+    width: 100%;
+    background-color: #fff;
+    position: fixed;
+  }
+
+  .navbar {
+    height: 72px;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: space-around;
+    width: 250px;
+  }
+
+  /* headline section */
+  .headline {
+    height: 100vh;
+    background-image: url(./src/assets/img/desktop-head-back.svg);
+    background-size: contain;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Please know that you shouldn't fix the height as well, instead, you can control how the container looks using `margins` and `paddings`, this will help you with flexibility as well, kindly fix this. 😁